src/pages/_app.page.tsx

```diff
@@ -1,7 +1,7 @@
+import { Urbanist } from '@next/font/google';
```

Comment:
Why this change?

<img width="1120" alt="Screenshot 2023-05-17 at 11 50 23" src="https://github.com/contentful/template-blog-webapp-nextjs/assets/103024358/47d20f71-1b3c-4a39-9d6d-3dfe83c79ad5">

Reply:
I changed the import and pushed again.